compiler/common-artifacts/src/TestDataGenerator.cpp

```diff
@@ -74,6 +76,19 @@ template <> void geneate_random_data<bool>(std::mt19937 &gen, void *data, uint32
   }
 }
 
+template <typename T>
+void geneate_random_range(void *data, uint32_t size, int32_t range_min, int32_t range_max)
+{
+  for (uint32_t i = 0; i < size; i++)
+  {
+    // +1 will make value of [range_min, range_max]
+    int32_t range = range_max - range_min + 1;
+    int32_t value = (rand() % range) + range_min;
+    static_cast<T *>(data)[i] = static_cast<T>(value);
+  }
+  std::cout << std::endl;
```

Comment:
This is intended code? @.@

Reply:
Ah, for debuging, not removed -_-;
Thanks!